managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerFactoryImpl.java

```diff
@@ -177,7 +175,7 @@ private ManagedLedgerFactoryImpl(MetadataStore metadataStore,
         this.mbean = new ManagedLedgerFactoryMBeanImpl(this);
         this.entryCacheManager = new EntryCacheManager(this);
         this.statsTask = scheduledExecutor.scheduleAtFixedRate(this::refreshStats,
-                0, StatsPeriodSeconds, TimeUnit.SECONDS);
+                config.getStatsPeriodSeconds(), config.getStatsPeriodSeconds(), TimeUnit.SECONDS);
```

Comment:
This is a little behavior change.
Before this change the stats were computed asap, now we have 1 minute happy.

What about keeping the previous behavior?

Reply:
Ok, I will change back to the previous one.